client/state/reducer.js

```diff
@@ -28,6 +29,7 @@ import userSettings from './user-settings/reducer';
 // Please do not add to this list. See #39261 and p4TIVU-9lM-p2 for more details.
 const reducers = {
 	currentUser,
+	currencyCode,
```

Comment:
Nit: this could be moved above if we wanted to keep these alphabetical.

Reply:
Alphabetized in abde221 :+1: